api/src/main/kotlin/edu/wgu/osmt/api/model/ApiKeyword.kt

```diff
@@ -8,32 +8,34 @@ import edu.wgu.osmt.keyword.KeywordTypeEnum
 
 @JsonInclude(JsonInclude.Include.ALWAYS)
 class ApiKeyword(
-    private val keyword: Keyword,
-    private val totalSkills: Long?,
```

Comment:
Since we aren't going to allow deletion of NamedReferences (Keywords) if they are still attached to an RSD, providing the caller with the number of associated RSDs would still be valuable.

Reply:
yeah that can be useful later, RN we are passing that flag(if a RSD is using that keyword or not) as a message in the result object? do you want me to keep it for further use?